src/classes/lyrics.ts

```diff
@@ -7,6 +7,16 @@ export type LyricsVerse = {
     content: string[];
 }
 
+let printLyrics = null as Lyrics | null;
+
+export function getPrintLyrics(): Lyrics | null{
+    return printLyrics;
+}
+
+export function setPrintLyrics(lyrics: Lyrics | null){
+    printLyrics = lyrics;
+}
+
```

Comment:
This can be written as 

```typescript
import { ref } from "vue"

export const printLyrics = ref(null as Lyrics | null)
```

Then used directly in components without get and set functions.